ci-operator/step-registry/baremetal/lab/ipi/install/baremetal-lab-ipi-install-commands.sh

```diff
@@ -188,6 +188,10 @@ cp "${INSTALL_DIR}/metadata.json" "${SHARED_DIR}/"
 cp "${INSTALL_DIR}/auth/kubeconfig" "${SHARED_DIR}/"
 cp "${INSTALL_DIR}/auth/kubeadmin-password" "${SHARED_DIR}/"
 
+# Creating file straight into $SHARED_DIR is not 100% reliable because of propagation issues (author guessing)
+oinst coreos print-stream-json > "${INSTALL_DIR}/coreos-stream.json"
+cp "${INSTALL_DIR}/coreos-stream.json" "${SHARED_DIR}/"
```

Comment:
Do we need this both in the INSTALL_DIR and SHARED_DIR or can we run this as in:

```suggestion
oinst coreos print-stream-json > "${SHARED_DIR}/coreos-stream.json"
```